src/main/resources/application.yml

```diff
@@ -41,4 +38,7 @@ management:
         enabled: true
 
 
-batch-size: 200
\ No newline at end of file
+mode: MULTIPLE
```

Comment:
Multiple here - is this for a test?

Reply:
it's the default, could be SINGULAR but it's set in the environment anyway